### PR TITLE
Refactor Configuration reads according to eventhub

### PR DESCRIPTION
**Pull request by tharindu1st**

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java [hunk 1]
```diff
@@ -6851,7 +6852,7 @@ private void publishKeyTemplateEvent(String templateValue, String state) {
 
 
         if (throttleProperties.getDataPublisher() != null && throttleProperties.getDataPublisher().isEnabled()) {
-            APIUtil.publishEvent(APIConstants.BLOCKING_EVENT_PUBLISHER, Collections.EMPTY_MAP, keyTemplateMessage);
+            APIUtil.publishEventToTrafficManager(Collections.EMPTY_MAP, keyTemplateMessage);
```
Comment (isharac) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java:6855]: TrafficManager or EventHub?

File: features/apimgt/org.wso2.carbon.apimgt.internal.service.feature/pom.xml [hunk 2]
```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+ ~ Copyright (c) 2010-2011, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (isharac) [inline features/apimgt/org.wso2.carbon.apimgt.internal.service.feature/pom.xml:3]: old license header

File: features/apimgt/org.wso2.carbon.apimgt.internal.service.feature/pom.xml [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+                                <artifactItem>
+                                    <groupId>org.wso2.carbon.apimgt</groupId>
+                                    <artifactId>org.wso2.carbon.apimgt.internal.service</artifactId>
+                                    <overWrite>true</overWrite>
+                                    <type>war</type>
+                                    <outputDirectory>${basedir}/src/main/resources/</outputDirectory>
+                                    <destFileName>internal#data#v1.war</destFileName>
+                                </artifactItem>
+                            </artifactItems>
+                            <outputDirectory>${project.build.directory}/external/libs</outputDirectory>
```
Comment (isharac) [inline features/apimgt/org.wso2.carbon.apimgt.internal.service.feature/pom.xml:109]: shouldn't this be copied to web apps?

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java (hunk 1 above)
Comment (tharindu1st) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java:6855]: this is correct

File: components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/jwt/RevokedJWTTokensRetriever.java [hunk 4]
```diff
@@ -62,13 +63,10 @@ public void run() {
 
         try {
             // The resource resides in the throttle web app. Hence reading throttle configs
```
Comment (isharac) [inline components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/jwt/RevokedJWTTokensRetriever.java:65]: we don't need this comment now

**Pull request merged**